getting_started.md

```diff
@@ -0,0 +1,109 @@
+# Getting Started
+
+## Workspace set up and installation
+To get started you will have to make a few decisions and install the repository along with its dependencies. Specifically:
+1. Decide if you would like to develop your submission in either Pytorch or Jax.
+2. Set up your workstation or VM. We recommend to use a setup similar to the competition scoring hardware which consists out of 8 V100 GPUs, 350 GB in RAM and 2 TB in storag for dataset storage. For recommendations on setting up your own Cloud VM see [here](https://github.com/mlcommons/algorithmic-efficiency/blob/main/docker/README.md#gcp-integration).
+4. Clone the algorithmic-efficiency repository and make sure you have installed the dependencies. To install the dependencies we recommend either:
+    1. Installing in a python virtual environment as described in the [Installation section in the README](https://github.com/mlcommons/algorithmic-efficiency/blob/main/README.md) instructions.
+    2. Use a Docker container with dependencies installed. This option will guarantee that you're code is running on the same CUDA, cudnn, and python dependencies as the competition scoring environment. See the [Docker README](https://github.com/mlcommons/algorithmic-efficiency/blob/main/docker/README.md) for instructions on the Docker workflow.
+
+## Download the data
+The workloads in this benchmark use 6 different datasets across 8 workloads. You may choose to download some or all of the datasets as you are developing your submission, but your submission will be scored across all 8 workloads. For instructions on obtaining and setting up the datasets see [datasets/README](https://github.com/mlcommons/algorithmic-efficiency/blob/main/datasets/README.md#dataset-setup).
+
+
+## Develop your submission
+To develop a submission you will write a python module containing your optimizer algorithm. Your optimizer must implement a set of predefined API methods for the initialization and update steps.
+
+### Set up your directory structure (Optional)
+
+1. Make a submissions subdirectory to store your submission modules e.g. `algorithmic-effiency/my_submissions`.
```

Comment:
Can you make this `algorithmic-efficiency/submissions/my_submissions` ? And delete the `2.` point below and say this is the recommended approach?

There's a longer issue of how we are going to link to/manage submitter code. we don't want to repeat `tf.contrib`'s nightmare where they included tons of random code with sprawling dependencies, which while convenient at first for users eventually made it unmanageable with stability issues. but we may want to include some more examples in an `algorithmic-efficiency/submissions/` dir, and maybe include the winning/top submissions in there using [git submodules](https://gist.github.com/gitaarik/8735255)?

But regardless, we should make an empty `algorithmic-efficiency/submissions/` dir for now, so that submitters can have a consistent place to keep their code.

Reply:
Ok that sounds good to me.